software/utils/rpi_config/README.md

```diff
@@ -1,42 +1,64 @@
+* Flash the SD card with [Raspberry Pi OS](https://www.raspberrypi.com/software/operating-systems/). This setup has most recently been tested with `Raspberry Pi OS with Desktop (64-bit) / January 28, 2022 release`.
+* If you have a keyboard and mouse attached, boot into the RasPi directly and configure it as follows:
+  * username (mandatory): `admin` - system should boot to this user by default, otherwise shortcuts may not work
+  * password (recommended): `respira` - will need this for `sudo` tasks, so don't forget it
+  * hostname (recommended): `rw-ventilator` - could make it something else unique if you have multiple prototypes on your network
+  * enable *ssh* and set up credentials to access the machine remotely
+* If you cannot or don't want to attach keyboard and mouse to your Raspberry Pi, you can configure the machine remotely via *ssh* as described our guide [here](headless_pi.md).
+* If configured as above, you should be able to *ssh* into the RasPi from another machine by running `ssh admin@rw-ventilator` (or whatever host name you gave it).
+* Otherwise, you can find out*RPi*'s IP address with `ifconfig` and then you can reach it as `ssh admin@IP_ADDRESS`.
+* Regardless of the approach taken above, you should now be able to run `sudo raspi-config` and apply the following settings:
+  * splash screen: `off`
+  * screen blanking: `off`
+  * audio output: `HDMI`
+  * Serial: `shell off`, `interface on`
+  * Locale: `[your locale]`
+  * Time zone: `[your time zone]`
+  * Boot into Desktop with automatic login
 
-Just copy and paste this dangerous script into the console:
+## Automatic setup
 
+With the above configured, the next step should be pretty automatic. You can still use a stand-alone RasPi without all the hardware.
+
+Use `ssh` or open up a terminal on the RasPi and just copy and paste this script into a terminal:
+
+```shell
+# \TODO: change command to point to master before merging!!!
+
+bash <(wget -qO- https://raw.githubusercontent.com/RespiraWorks/Ventilator/issue_1180_cmake_build_on_rpi/software/utils/rpi_config/bootstrap.sh)
 ```
-bash <(wget -qO- https://raw.githubusercontent.com/RespiraWorks/Ventilator/master/software/utils/rpi_config/bootstrap.sh)
-```
 
-Follow the onscreen directions.
+Follow the onscreen directions. When the initial installation is complete, the system will reboot.
+
+This is now a good time to:
+* transfer the SD card to the target machine
+* make sure you have a USB connection from the PI to the Nucleo so that it can update controller firmware
+* if you have `ssh` access, the rest should work without the need for keyboard and mouse attached to the PI
+
+### Final configuration and initial software deployment
 
-When the initial install is complete, the system will reboot again.
+When the machine boots, you should see an icon that says `Ventilator update` on the desktop. Either double-click or use the touch screen to run this shortcut.
```

Comment:
When called like this, the script may fail silently (the terminal closes down when the error happens and we can't see the error message). Maybe indicate that we need to check that the terminal doesn't close itself and prompts
```shell
Installation complete. Please check that this script terminated without errors.
Press any key to continue.
```

And indicate to run the scripts manually (via ssh?) to diagnose the error.